Add actions

quswadress opened the pull request:
Add github actions workflow. Closes #8 

TODO:
- [x] Create linux exe
- [x] Create windows exe
- [x] Test it. (@Danilado didn't turn on Actions) 😞
- [x] Automatically create release on main push

quswadress committed:
Create build.yml

Danilado commented:
Turned on Actions

quswadress committed:
Delete .gitlab-ci.yml

quswadress committed:
Test action

quswadress committed:
Trying to fix action. Try 1/∞

quswadress committed:
Trying to fix action. Try 2/∞

quswadress committed:
Trying to fix action. Try 4/∞

quswadress committed:
Trying to fix action. Try 3/∞

quswadress committed:
Trying to fix action. Try 5/∞

quswadress committed:
Trying to fix action. Try 6/∞

quswadress committed:
Trying to fix action. Try 7/∞. Added pyproject.toml

quswadress committed:
Trying to fix action. Try 8/∞.

quswadress committed:
Fix naming and remove win-2019

Danilado commented on README.md:
Should also change readme-ru accordingly

quswadress replied:
I've merged the develop and fixed the conflicts.

quswadress committed:
Move version to a single file

quswadress committed:
Trying to add automatic release. Try 1/∞.

quswadress committed:
Trying to add automatic release. Try 2/∞.

quswadress committed:
Trying to add automatic release. Try 3/∞.

quswadress committed:
Added auto release.

Now commits in main automatically create a release and tag with the version specified in the file.
[no ci]

quswadress committed:
Merge branch 'develop' into 8-change-cicd

# Conflicts:
#	README.md

quswadress committed:
Added instructions in the README for downloading from the release.
